.github/workflows/go.yml

```diff
@@ -10,6 +10,9 @@ jobs:
   build:
     name: Build
     runs-on: ubuntu-latest
+    strategy:
+        matrix:
```

Comment:
Should we also add `MONGODB_FLAVOR` to the matrix to test with PSMDB?

Reply:
We are not using any special Percona implemented feature so, right now, there is no need to test with Percona Server for Mongodb and MongoDB itself.

Reply:
I think we should test it anyway.

Reply:
Now we have `TEST_MONGODB_FLAVOR` defined in Makefile that is not used?